### ISSUE TITLE
This traverser does not support merging org.apache.tinkerpop.gremlin.process.traversal.traverser.O_Traverser

### ISSUE THREAD

**Issue by ericpeters0n** [awslabs/dynamodb-titan-storage-backend#37]
When running step 12 of the example

```
:> g.V().has('comic-book', 'AVF 4').in('appeared').values('character').order()
```

**Comment by ericpeters0n** [comment]
Additionally, the outputs of the example queries preceding simply give IDs, rather than anything readable. e.g.

```
gremlin> :> g.V().as('character').has('weapon', without('shield','claws')).out('appeared').as('comic-book').select('character','comic-book')
==>{character=v[37080], comic-book=v[57496]}
==>{character=v[37080], comic-book=v[57496]}
==>{character=v[37080], comic-book=v[57536]}
==>{character=v[37080], comic-book=v[57536]}
...
```
 Is this intentional/correct?

**Comment by amcp** [comment]
Fixed in e3d9da28f8302eba77859646899694300a6a1551

**Comment by amcp** [comment]
I was not able to reproduce your error `This traverser does not support merging org.apache.tinkerpop.gremlin.process.traversal.traverser.O_Traverser`. It may have been fixed so if it becomes an issue again please open this issue again.
